src/content/docs/iast/troubleshooting/unable-to-see-application.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Unable to see application
+tags:
+    - IAST
+    - New Relic IAST
+    - Troubleshooting
+metaDescription: Unable to find my application in the New Relic UI. 
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+I don't see my application in the New Relic UI.
+
+
+
+## Solution
+
+    If you don't see your application in <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST > All Tests**</DNT>, check the following:
```

Comment:
The bullets here are not parallel. Please reach each as a continuation of "...check the following"